core/templates/dev/head/admin/admin.html

```diff
@@ -11,93 +11,249 @@
+                                  </div>
+                                {% endif %}
+                              </div>
+                            </a>
+                            <ul class="dropdown-menu ng-cloak oppia-navbar-dropdown" role="menu" ng-mouseover="onMouseoverProfilePictureOrDropdown($event)" ng-mouseleave="onMouseoutProfilePictureOrDropdown($event)" ng-show="profileDropdownIsActive">
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/profile/{{username}}">
+                                  <strong>{{username}}</strong>
+                                </a>
+                              </li>
+                              <hr class="oppia-top-right-menu-item-separator">
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/my_explorations">
+                                  My Explorations
+                                </a>
+                              </li>
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/notifications_dashboard">
+                                  Notifications
+                                  <span ng-if="numUnseenNotifications > 0">
+                                    (<[numUnseenNotifications]>)
+                                  </span>
+                                </a>
+                              </li>
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/preferences">Preferences</a>
+                              </li>
+
+                              {% if is_moderator %}
+                                <li>
+                                  <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/moderator" target="_blank">
+                                    Moderator Page
+                                  </a>
+                                </li>
+                              {% endif %}
+                              {% if is_super_admin %}
+                                <li>
+                                  <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/admin" target="_blank">
+                                    Admin Page
+                                  </a>
+                                </li>
+                              {% endif %}
+                              <hr class="oppia-top-right-menu-item-separator">
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="{{logout_url}}">Logout</a>
+                              </li>
+                            </ul>
+                          </li>
+                        {% else %}
+                          <li class="dropdown oppia-clickable-dropdown pull-right">
```

Comment:
This page shouldn't be shown to anyone who isn't signed in, so this clause isn't needed